test/support/regression_line_pos_iterator.cpp

```diff
@@ -124,23 +124,64 @@ void testLRorCR_foo_bar(std::string const& line_break) {
     test(input, validations);
 }
 
+// Multiple Empty Lines
+
+void  testMultipleEmptyLines(std::string const& line_break) {
+    std::string const input = "foo" + line_break + "bar"
+                            + line_break + line_break + "foo"
+                            + line_break + line_break + line_break + "bar";
+    validations const validations = boost::assign::list_of<validation>
+        (1,1,"foo")(1,2,"foo")(1,3,"foo")(1,4,"foo")
+        (2,1,"bar")(2,2,"bar")(2,3,"bar")(2,4,"bar")
+        (3,1,"")
+        (4,1,"foo")(4,2,"foo")(4,3,"foo")(4,4,"foo")
+        (5,1,"")
+        (6,1,"")
+        (7,1,"bar")(7,2,"bar")(7,3,"bar");
+    test(input, validations);
+}
+
+void  testMultipleEmptyLinesRN(std::string const& line_break) {
+    std::string const input = "foo" + line_break + "bar"
+                            + line_break + line_break + "foo"
+                            + line_break + line_break + line_break + "bar";
+    validations const validations = boost::assign::list_of<validation>
+        (1,1,"foo")(1,2,"foo")(1,3,"foo")(1,4,"foo")
+        (2,1,"")(2,1,"bar")(2,2,"bar")(2,3,"bar")(2,4,"bar")
+        (3,1,"")(3,1,"")
+        (4,1,"")(4,1,"foo")(4,2,"foo")(4,3,"foo")(4,4,"foo")
+        (5,1,"")(5,1,"")
+        (6,1,"")(6,1,"")
+        (7,1,"")(7,1,"bar")(7,2,"bar")(7,3,"bar");
```

Comment:
This doesn't look right to me. Am I missing something?

Reply:
well, at least the line got counted. 
the position repeats itself, that is a bit strange, but i can look into it...

Reply:
Nevermind, that's happening because of iterator is inside multichar newline